music/views.py

```diff
@@ -47,3 +58,71 @@ def get(self, request, media_id):
                                      status=200, content_type='audio/mp3')
         resp['Cache-Control'] = 'no-cache'
         return resp
+
+class MainView(View):
+
+    def get(self, request):
+
+        range_list = [1,7,13]
+
+        if request.GET :
+            range_list = request.GET.getlist(
+                'collection_id'
+            )
+
+        collection = Collection.objects.prefetch_related(
+            'playlist_set'
+        )
```

Comment:
prefetch_related 에 인자가 하나이고 한줄에 써도 무방할 것 같아요.
```suggestion
        collection = Collection.objects.prefetch_related('playlist_set')
```

Reply:
코드 복잡도가 낮을땐, 한줄로 쓰도록 하겠습니다 ㅎ